setup.py

```diff
@@ -26,8 +26,11 @@
 
 EXTRAS_REQUIRE = {
     "neural-compressor": ["neural-compressor>=2.0.0", "onnx", "onnxruntime"],
-    "openvino": ["openvino>=2022.3.0", "onnx", "onnxruntime"],
-    "nncf": ["nncf"],
+    "openvino": ["openvino>=2022.3.0", "transformers>=4.20.0", "onnx", "onnxruntime"],
```

Comment:
```suggestion
    "openvino": ["openvino>=2022.3.0", "onnx", "onnxruntime"],
```

Reply:
revised in [489022f](https://github.com/huggingface/optimum-intel/pull/150/commits/489022fcf259866a8b9214f385f98ccef58b4beb)